### PR TITLE
Fix #2185 make wrong command return non-zero value

### LINKED ISSUE #2185: Inconsistent cli document about wrong command execution will return non-zero value
## Description

Run any wrong command, then run `echo $?`, will return 0.
## Describe How to Reproduce

```
peer dsdsd
echo $?
```

### PR DESCRIPTION
**Pull request by GrapeBaBa**
## Description

Uncomment os.Exit(1) in main.go and remove main_test.go
## Motivation and Context

Fixes #2185 
1.Now after run wrong command will return 0 value and it is inconsistent with the document https://github.com/hyperledger/fabric/blob/master/docs/API/CoreAPI.md#cli
2.Now the main_test.go actually will run a wrong command `peer -test.timeout=20`, that is not a good test
## How Has This Been Tested?

make checks successfully
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [x] I have either added documentation to cover my changes or this change requires no new documentation.
- [x] I have either added unit tests to cover my changes or this change requires no new tests.
- [x] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

Signed-off-by:Kai Chen ckaiwh@cn.ibm.com

### CODE REVIEW

File: peer/main_test.go [hunk 1]
```diff
@@ -39,5 +39,5 @@ func TestSystem(t *testing.T) {
 	// if *systemTest {
 	// 	main()
 	// }
-	main()
+	//main()
```
Comment (gabre) [inline peer/main_test.go]: This is "dead" code (commented). Should we preserve dead code? (Git implicitly preserves it in the history)

**Comment by gabre** [comment]
Just a nit: maybe adding some motivation (why this fixes that, OK I know it is trivial) would be good for documentational purposes.

File: peer/main_test.go (hunk 1 above)
Comment (GrapeBaBa) [inline peer/main_test.go]: yes, i'd like to remove main_test.go. what about your thoughts?

**Comment by srderson** [comment]
LGTM, thanks

**Pull request merged**